src/utils/py/daos_api.py

```diff
@@ -1876,13 +1876,15 @@ def create(self, coh, epoch):
     #  snapshots and the epochs and names lists. See description of
     #  daos_cont_list_snap in src/include/daos_api.h. This must be done for
     #  DAOS-1336 Verify container snapshot info.
-    def list(self, coh):
+    def list(self, coh, epoch=None):
```

Comment:
From the C API:
  * \param[out]  epochs  preallocated array of epochs to store snapshots.

So this is actually where we return the snapshot list.